doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -7703,3 +7703,29 @@ Parameters:
 
 ``per_page`` Number of results returned per page.
 
+
+Customization File Contents
+---------------------------
+
+The Customization API is used to retrieve the analytics-code.html as well as other customization file contents. See also :ref:`web-analytics-code` in the Configuration section of the Installation Guide.
+
+The content is returned in type="text/html; charset=UTF-8"
+
+Valid types are "homePage", "header", "footer", "style", "analytics", and "logo".
```

Comment:
We should probably link to...

```
:ref:`Branding Your Installation`
```

... so people can know what these are (header, logo, etc.).

And from "branding" and "web analytics" we should link back to these API docs so people know it's possible to download these files via API.